lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -586,6 +613,18 @@ private void sendAuditEvent(
                         new AuditRestrictedDeviceInformation(deviceInformation)));
     }
 
+    private void sendAuditEventWithPreviousIpvSessionId(
+            AuditEventTypes auditEventTypes,
+            AuditEventUser auditEventUser,
+            String previousIpvSessionId) {
+        auditService.sendAuditEvent(
+                AuditEvent.createWithoutDeviceInformation(
```

Comment:
@blakeyp would we want to add device information to this event?

Reply:
Yep we should have device information if it's a user-initiated rather than purely backend process

Reply:
I was copying the existing DCMAW audit events but will add in the device information.